tests/analyses/test_api.py

```diff
@@ -14,8 +14,13 @@
 
 
 @pytest.fixture
-def files(tmp_path):
-    path = Path.cwd() / "tests" / "test_files" / "aodp" / "reference.fa"
+def test_files_path() -> Path:
```

Comment:
There is already a fixture in `virtool.tests.fixtures.core` called `test_files_path` that does the same thing.